src/conditionals/simple_programs_with_decision_points/medium/q6/SquareUp.java

```diff
@@ -0,0 +1,24 @@
+package conditionals.simple_programs_with_decision_points.medium.q6;
+/*
+Write a program to take in the measurements of length and width of a rectangle to determine if it is square or not.
+Sample Output:
+	Enter the length: 5
+	Enter the width: 6
+	Just another rectangle…
+ */
```

Comment:
Perhaps change the sample output to double inputs instead of integer? I know it's a common occurrence with students to assume integers unless told it should be a double